eventually/customprofile/migrations/0002_added_relation.py

```diff
@@ -1,22 +1,28 @@
 # -*- coding: utf-8 -*-
-# Generated by Django 1.11.6 on 2017-11-09 13:26
+# Generated by Django 1.11.6 on 2017-11-21 10:58
 from __future__ import unicode_literals
 
+from django.conf import settings
 from django.db import migrations, models
 import django.db.models.deletion
 
 
 class Migration(migrations.Migration):
 
     dependencies = [
-        ('authentication', '0001_initial'),
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
```

Comment:
seems it may be the error. Check if this line could be here.

Reply:
done